tsconfig.compilerOption.json

```diff
@@ -12,7 +12,6 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true,
```

Comment:
I'm guessing that this'll raise a few eyebrows—I did it because I combined all the if-elses into a memoized var, `status`. To make Empty and Updating show Success when they're undefined, those cases had to fall through. Happy to do it differently, but thought I'd get feedback on this first.

Reply:
Scratched the fall through for now.